frontend/app/pages/project/[slug]/repository-group/[groupSlug]/community-voice.vue

```diff
@@ -0,0 +1,16 @@
+<!--
+Copyright (c) 2025 The Linux Foundation and each contributor.
+SPDX-License-Identifier: MIT
+-->
+<template>
+  <lfx-community-voice-view />
+</template>
+
+<script setup lang="ts">
+import LfxCommunityVoiceView from '~/components/modules/project/views/community-voice.vue';
+
+// Prevent search engines from indexing this page
+useSeoMeta({
+  robots: 'noindex, nofollow',
```

Comment:
Community voice is project only so i dont think its needed to be there for repository groups and repositories. Check with jonathan if we hide it when users are filtering